packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -145,12 +145,6 @@ export default class StatsAnalyzer extends EventsScope {
    */
   startAnalyzer(peerConnection) {
     this.peerConnection = peerConnection;
-    this.peerConnection.onconnectionstatechange = () => {
-    // Automatically stop once the connection state is closed
-      if (this.peerConnection.connectionState === PEER_CONNECTION_STATE.CLOSED) {
-        this.stopAnalyzer();
```

Comment:
Does this need to be moved elsewhere?

Reply:
still need to figure out a best place. have added it in 

```
MeetingUtil.cleanUp = (meeting) => {
  // make sure we send last metrics before we close the peerconnection
  if (meeting.statsAnalyzer) meeting.statsAnalyzer.stopAnalyzer();
```

If we still see a bug where the status does not stop we can have a look